torch/ao/quantization/utils.py

```diff
@@ -326,10 +326,6 @@ def calculate_qmin_qmax(quant_min: int, quant_max: int, has_customized_qrange: b
             assert (
                 0 < qrange_len <= 2**31
             ), "quantization range should be positive and not exceed the maximum bit range (=4294967296)."
-        if dtype == torch.qint8:
-            quant_min, quant_max = -qrange_len // 2, qrange_len // 2 - 1
-        else:
-            quant_min, quant_max = 0, qrange_len - 1
```

Comment:
I think you can remove the next line as well

Reply:
do you mean remove 'reduce_range' if statement?

Reply:
yes, L333-L334

Reply:
if we remove it, how to handle the qmin/max if user wanna use has_customized_qrange with reduce_range flag?

Reply:
it would break several test cases if we remove it.

Reply:
user should set the range correctly in the beginning directly I think?